Source/WebCore/platform/ScrollableArea.h

```diff
@@ -31,6 +31,7 @@
 #include "ScrollSnapOffsetsInfo.h"
 #include "ScrollTypes.h"
 #include "Scrollbar.h"
+#include "ScrollbarGutter.h"
```

Comment:
It’s unfortunate we have to add a header include. Lots of files include `ScollableArea.h` and this will slow down the build. If the default implementation of the virtual function was not inlined in the header then we could just use a forward declaration here and include the header only where it’s used.

Reply:
Will make that change now